kyverno/metadata.csv

```diff
@@ -1 +1,71 @@
+kyverno.http.requests.duration.seconds.count,count,,,,The count aggregation shows the number of observations in the HTTP requests latency histogram,0,kyverno,kyverno http requests duration seconds count,,
+kyverno.http.requests.duration.seconds.sum,count,,second,,The sum aggregation of the HTTP requests represents the total amount of seconds spent on HTTP requests,0,kyverno,kyverno http requests duration seconds sum,,
+kyverno.policy.changes.count,count,,,,"Metric used to track the history of all Kyverno policy-related changes such as policy creations, updates, and deletions",0,kyverno,kyverno policy changes count,,
+kyverno.policy.execution.duration.seconds.bucket,count,,,,The bucket aggregation shows the number of observations within each distribution bucket in the Policy rule execution latency histogram,0,kyverno,kyverno policy execution duration seconds bucket,,
+kyverno.policy.execution.duration.seconds.count,count,,,,The count aggregation shows the number of observations in the policy rule execution latency histogram,0,kyverno,kyverno policy execution duration seconds count,,
+kyverno.policy.execution.duration.seconds.sum,count,,second,,The sum aggregation of the policy rule execution represents the total amount of seconds spent on the the execution/processing of the individual rules whenever they evaluate incoming resource requests or execute background scans,0,kyverno,kyverno policy execution duration seconds sum,,
+kyverno.policy.results.count,count,,,,Metric used to track the results associated with the rules executing as a part of incoming resource requests and even background scans. This metric can be further aggregated to track policy-level results as well,0,kyverno,kyverno policy results count,,
+kyverno.policy.rule.info.count,count,,,,Number of policies as well as rules present in the cluster which are currently active and even the ones which are not currently active but were created in the past,0,kyverno,kyverno policy rule info count,,
+kyverno.process.cpu.seconds.count,count,,second,,The total user and system CPU time spent in seconds in the Kyverno instance,0,kyverno,kyverno process cpu seconds,,
+kyverno.process.max_fds,gauge,,,,The maximum number of open file descriptors in the Kyverno instance,0,kyverno,kyverno process max_fds,,
+kyverno.process.open_fds,gauge,,,,The number of open file descriptors in the Kyverno instance,0,kyverno,kyverno process open_fds,,
+kyverno.process.resident_memory.bytes,gauge,,byte,,The resident memory size in bytes in the Kyverno instance,0,kyverno,kyverno process resident_memory bytes,,
+kyverno.process.start_time.seconds,gauge,,second,,The start time of the process since unix epoch in seconds in the Kyverno instance,0,kyverno,kyverno process start_time seconds,,
```

Comment:
```suggestion
kyverno.process.start_time.seconds,gauge,,second,,The start time of the process since the Unix epoch in seconds in the Kyverno instance,0,kyverno,kyverno process start_time seconds,,
```